fabric-data-generation-api-v1/src/testmod/java/net/fabricmc/fabric/test/datagen/DataGeneratorTestEntrypoint.java

```diff
@@ -110,6 +119,36 @@ protected void generateRecipes(Consumer<RecipeJsonProvider> exporter) {
 		}
 	}
 
+	private static class ExistingEnglishLangProvider extends FabricLanguageProvider {
+		private ExistingEnglishLangProvider(FabricDataGenerator dataGenerator) {
+			super(dataGenerator);
+		}
+
+		@Override
+		public void generateLanguages(LanguageConsumer languageConsumer) {
+			languageConsumer.addLanguage(SIMPLE_BLOCK, "Simple Block");
+
+			try {
+				languageConsumer.addLanguage(dataGenerator.getModContainer().getRootPaths().get(1).resolve("assets/testmod/lang/en_us.json"));
```

Comment:
Use `findPath` https://github.com/FabricMC/fabric-loader/blob/master/src/main/java/net/fabricmc/loader/api/ModContainer.java#L50-L61

What you have can and will explode badly.